docs: Document the basic single-Email cloning workflow (#822)
mautic/user-documentation: docs/channels/emails.rst

Reviewer: ```suggestion
#. Click the down arrow button next to **Schedule** to open the **Options** menu.
```

The rendered preview on the Email details page reloads automatically when you switch between variants and translations. For more information, see :ref:`Email content preview`.

Cloning Emails

> ==============

Author: Confirmed the basic `clone` action is a standard `templateButton` in the Email detail view's actions dropdown (down arrow next to Schedule), rendered via `page_actions.html.twig` and gated on the `email:emails:create` permission — this grounds the access path and the permission note.

Source: https://github.com/mautic/mautic/blob/5.x/app/bundles/EmailBundle/Resources/views/Email/details.html.twig

> Cloning Emails
> ==============

Reviewer: ```suggestion
.. vale off

Cloning Emails
==============

.. vale on
```

Cloning an Email creates an editable copy that you can adjust and save as a new Email. This is useful when you want to reuse the content and settings of an existing Email as a starting point.

> To clone an Email:

Reviewer: There are 2 ways to clone an email:

1. By clicking the three-dots icon near the email name when you visit the Email main page. (don't say 'main' here. Say something else like overview or listing).
2. In the email detail page (as you mentioned here)

#. Open the Email you want to copy to view its details.
#. Click the down arrow button next to **Schedule** on the Email detail view.
#. Select **Clone**.

> Mautic opens the copy in the Email editor with the original content and settings prefilled. Adjust the copy as needed, then save it to create the new Email.

Author: `cloneAction()` clones the entity and builds an editable Email form prefilled with the original content and settings (`clone $emailEntity` then `createForm`), saving a new Email on submit — this grounds the "opens the copy in the Email editor ... then save it to create the new Email" behavior and the create-permission gate (`isGranted('email:emails:create')`).

Source: https://github.com/mautic/mautic/blob/5.x/app/bundles/EmailBundle/Controller/EmailController.php

.. note::

   Cloning requires permission to create Emails. If you can't create Emails, the **Clone** action doesn't appear.

.. vale off

> Clone with translations and variants

Reviewer: As we're adding the "Cloning Emails" section, this would be suitable as its subsection.

.. vale off

Cloning Emails
==============

.. vale on

Cloning an Email creates an editable copy that you can adjust and save as a new Email. This is useful when you want to reuse the content and settings of an existing Email as a starting point.

There are two ways to clone an Email:

* From the Email listing, click the three-dots icon next to the Email's name, then select **Clone**.

> * From the Email detail view, click the down arrow button next to **Schedule**, then select **Clone**.

Reviewer: As we'll have images, we can break this for easy read. Add as well images, name them `email_overview_clone.png` and `email_clone.png`, and add descriptive alt text. Don't start alt text with "screenshot of" or things like that.
 
```suggestion
* **From the Email listing**:

  #. In the Email row, click the three-dots icon next to the checkbox to open the **Options** menu.
  #. Select **Clone**.
  
     |
  
     [email_overview_clone.png]
     
     |
  
* **From the Email detail view**:

  #. Click the down arrow button next to **Schedule**.
  #. Select **Clone**.
  
     |
     
     [email_clone.png]
     
     |
```

**Images:**

email_overview_clone.png

<img width="2020" height="970" alt="Image" src="https://github.com/user-attachments/assets/7e8bdcda-f5bc-4f48-b8bc-9168543c6c75" />

email_clone.png

<img width="1991" height="845" alt="Image" src="https://github.com/user-attachments/assets/29cfe18c-2c48-4237-b82e-2ec5e84bbbd1" />

Either way, Mautic opens the copy in the Email editor with the original content and settings pre-populated. Adjust the copy as needed, then save it to create the new Email.

.. note::

>    Cloning requires permission to create Emails. If you can't create Emails, the **Clone** action doesn't appear.

Reviewer: ```suggestion
   Cloning requires permission to create Emails. If you don't have the permission, the **Clone** option doesn't appear.
```

.. vale off

Cloning Emails
==============

.. vale on

Cloning an Email creates an editable copy that you can adjust and save as a new Email. This is useful when you want to reuse the content and settings of an existing Email as a starting point.

There are two ways to clone an Email:

* **From the Email listing**:

  #. In the Email row, click the three-dots icon next to the checkbox to open the **Options** menu.
  #. Select **Clone**.

     |

     .. image:: images/emails/email_overview_clone.png
        :width: 800
        :align: center
        :alt: Options menu open on an Email row in the Email listing, with Clone highlighted.

     |

* **From the Email detail view**:

>   #. Click the down arrow button next to **Schedule**.

Reviewer: ```suggestion
  #. Click the down arrow button next to **Schedule** to open the **Options** menu.
```
